KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -409,87 +409,14 @@ public void onEvent(IKalturaEvent event){
 			{
 				case STREAM_PUBLISHED:
 					streamEvent = (KalturaStreamEvent) event;
-					logger.info("Calling KalturaLiveManager onPublish");
-					onPublish(streamEvent.getEntryId(), streamEvent.getServerIndex(), streamEvent.getApplicationName());
```

Comment:
The only reason we were fetching it was to call the onPublish which was removed so no I don't see the reason for leaving the synchronize there or the retrieval of the entry from the wntryCache